website/docs/r/pki_secret_backend_config_cmpv2.html.md

```diff
@@ -72,8 +73,10 @@ The following arguments are supported:
 * `enable_sentinel_parsing` - (Optional) If set, parse out fields from the provided CSR making them available for Sentinel policies.
 
 * `enabled` - (Optional) Specifies whether CMPv2 is enabled.
-
+  
 * `audit_fields` - (Optional) Fields parsed from the CSR that appear in the audit and can be used by sentinel policies.
+  
+* `disabled_validations` - (Optional) A comma-separated list of validations not to perform on CMPv2 messages.
```

Comment:
Might be nice to mention what the (current) supported options are.

Reply:
I believe the norm isn't to specify a list here as it can fall out of sync with the server side of things, especially across revisions. 

Its the same reasoning the TFVP team have shied away from enforcing the valid values on the input schema as it causes end-user friction that we need to update and publish a new TFVP version every time something on the server changes